docs/hook/amp_validation_error.md

```diff
@@ -0,0 +1,14 @@
+## Hook `amp_validation_error`
+
+### Source
+
+:link: [includes/validation/class-amp-validation-manager.php:848](../../includes/validation/class-amp-validation-manager.php#L848)
+
+<details>
+<summary>Show Code</summary>
+
+```php
+$error = apply_filters( 'amp_validation_error', $error, compact( 'node' ) );
+```
```

Comment:
Something is missing here: the phpdoc!

The hook is:

https://github.com/ampproject/amp-wp/blob/914a5cfb3efc97853f33552cd7e4bf11d0212a59/includes/validation/class-amp-validation-manager.php#L827-L848

And yet it is generated as:

![image](https://user-images.githubusercontent.com/134745/92077511-6e8c5780-ed71-11ea-9047-c281202de183.png)

Where's the description and the parameters?

Reply:
Hook names should be self-explanatory and not need documentation! 😉

Reply:
It turns out I never completed the template for the individual hooks. I'll add a commit here...

Reply:
I pushed what I have so far. Descriptions for the hooks were added, and there's a distinction now between actions and filters, both in the way they are described as well as in an additional categorisation in their index page.

The parameters are still missing because there's something off in the object tree. The arguments that are attached to the hook have no name, so I can't fetch their corresponding docblock right now. Still working on this, but it might be more involved so it shouldn't block this improvement in case it should be merged for v2.0.1.